components/bl-pdfViewer-component/src/icons.js

```diff
@@ -0,0 +1,91 @@
+	      82.322,328.536,86.161,330,90,330s7.678-1.464,10.607-4.394l149.999-150.004c2.814-2.813,4.394-6.628,4.394-10.606
+	      C255,161.018,253.42,157.202,250.606,154.389z"
+      />
+    </svg>
+  );
+}
+
+export function PrintIcon() {
+  return (
+    <svg className="button-icon" viewBox="0 0 45 45">
+      <g>
+        <path d="M42.5,19.408H40V1.843c0-0.69-0.561-1.25-1.25-1.25H6.25C5.56,0.593,5,1.153,5,1.843v17.563H2.5
+    			c-1.381,0-2.5,1.119-2.5,2.5v20c0,1.381,1.119,2.5,2.5,2.5h40c1.381,0,2.5-1.119,2.5-2.5v-20C45,20.525,43.881,19.408,42.5,19.408z
+    			M32.531,38.094H12.468v-5h20.063V38.094z M37.5,19.408H35c-1.381,0-2.5,1.119-2.5,2.5v5h-20v-5c0-1.381-1.119-2.5-2.5-2.5H7.5
+    			V3.093h30V19.408z M32.5,8.792h-20c-0.69,0-1.25-0.56-1.25-1.25s0.56-1.25,1.25-1.25h20c0.689,0,1.25,0.56,1.25,1.25
+    			S33.189,8.792,32.5,8.792z M32.5,13.792h-20c-0.69,0-1.25-0.56-1.25-1.25s0.56-1.25,1.25-1.25h20c0.689,0,1.25,0.56,1.25,1.25
+    			S33.189,13.792,32.5,13.792z M32.5,18.792h-20c-0.69,0-1.25-0.56-1.25-1.25s0.56-1.25,1.25-1.25h20c0.689,0,1.25,0.56,1.25,1.25
+    			S33.189,18.792,32.5,18.792z"
+        />
+      </g>
+    </svg>
+  );
+}
+
+export function DownloadIcon() {
+  return (
+    <svg className="button-icon" fill="#ffffff" viewBox="0 0 472.615 472.615">
+      <g stroke-width="0"></g>
+      <g stroke-linecap="round" stroke-linejoin="round"></g>
+      <g>
+        <g>
+          <g>
+            <polygon points="354.462,5.804 354.462,88.586 437.337,88.592 "></polygon>
+          </g>
+        </g>
+        <g>
+          <g>
+            <polygon
+              points="68.923,433.23 68.923,49.231 29.538,49.231 29.538,472.615 393.846,472.615 393.846,433.23 "></polygon>
+          </g>
+        </g>
+        <g>
+          <g>
+            <path
+              d="M334.769,108.278V0H88.615v413.538h354.462V108.285L334.769,108.278z M226.46,196.897v-128h59.077v128h39.385 l-68.923,68.923l-68.923-68.923H226.46z M374.154,295.355c0,21.721-17.663,39.385-39.385,39.385H177.231 c-21.722,0-39.385-17.664-39.385-39.385v-78.769h19.692v78.769c0,10.855,8.837,19.692,19.692,19.692h157.538 c10.855,0,19.692-8.837,19.692-19.692v-78.769h19.692V295.355z"></path>
+          </g>
+        </g>
+      </g>
+    </svg>
+  );
+}
+
+export function PlusIcon() {
+  return (
+    <svg className="button-icon" viewBox="0 0 24 24" fill="none">
+      <g>
+        <path d="M5 12H19" stroke="#ffffff" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"></path>
+        <path d="M12 5L12 19" stroke="#ffffff" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"></path>
+      </g>
+    </svg>
+  );
+}
+
+export function MinusIcon() {
+  return (
+    <svg className="button-icon" viewBox="0 0 20 20" fill="none">
+      <g id="SVGRepo_bgCarrier"></g>
+      <g id="SVGRepo_tracerCarrier"></g>
+      <g id="SVGRepo_iconCarrier">
+        <path fill="#ffffff" d="M18 10a1 1 0 01-1 1H3a1 1 0 110-2h14a1 1 0 011 1z"></path>
```

Comment:
Some user in future ask to change colors of these icons. Is it possible now? If no could we add it in easy way(without properties)?